auth/src/main/java/org/openedx/auth/presentation/signin/compose/SignInView.kt

```diff
@@ -244,7 +244,7 @@ private fun AuthForm(
                 .fillMaxWidth()
                 .padding(top = 20.dp, bottom = 36.dp)
         ) {
-            if (state.isLogistrationEnabled.not()) {
+            if (state.isLogistrationEnabled.not() and state.isRegistrationEnabled) {
```

Comment:
Please change `and` to `&&` to keep style consistency.

Reply:
Changed